### DISCUSSION TITLE
Lazer keeps trying to force fullscreen when switching tabs while running on battery power

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20939] [Q&A]
### Type

Other

### Bug description

When switching tabs from lazer in windows 11, lazer will try to force fullscreen, fail, then try again. The only way to get out of this loop is to spam alt F4 as the screen is black. This does not happen with a power cord

I will try to elaborate on conditions later, but I have class now.

### Version

2022.1022.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9854322/database.log)
[input.log](https://github.com/ppy/osu/files/9854323/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9854324/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9854325/network.log)
[performance.log](https://github.com/ppy/osu/files/9854326/performance.log)
[performance-draw.log](https://github.com/ppy/osu/files/9854327/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9854328/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9854329/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9854330/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9854331/updater.log)

**Comment by peppy** [comment]
Can you explain what you mean by "switching tabs"?

**Comment by Doksha** [comment]
> Can you explain what you mean by "switching tabs"?

using the Alt+tab hotkey to switch to another window (to check discord or check the time for example)

this also happens when anything tries to force lazer out of fullscreen (such as popups for caps lock and num lock)

This results in a crash if you try to play a map in this state (by clicking enter)

**Comment by Doksha** [comment]
Setting the GPU to the dedicated GPU fixes the issue (windows would only select the integrated graphics when on battery power)
so the issue is with the integrated graphics.
